### ISSUE TITLE
Let it snow & PenguinMark: no way to mute background music

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#210]
Not sure if this will change the perf characteristics of the demos, but due to the background sound playing constantly, there should be a mute button to pass "audio control". (Usually causes issue with playing ove sreen readers)us

**Comment by alrra** [comment]
Notes:

  * `Narrator` turns the background volume a bit down.
  * Both demos have a (hidden) way to stop the music:

       * pressing the letter `p` for  [`Let It Snow`](https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/letitsnow/) (doesn't work in all browsers)
      * pressing the letter `m` for [`Penguin Mark`](https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/penguinmark/)

**Comment by alrra** [comment]
Closing this in favor of:

- [x] `Let It Snow` - https://github.com/MicrosoftEdge/Demos/pull/256
- [x] `Penguin Mark` - https://github.com/MicrosoftEdge/Demos/pull/269
